pom.xml

```diff
@@ -12,36 +12,72 @@
   </description>
   <developers>
     <developer>
-      <id>derka</id>
-      <name>Martin Derka</name>
-      <email>derka@google.com</email>
+      <id>garrettjonesgoogle</id>
+      <name>Garrett Jones</name>
+      <email>garrettjones@google.com</email>
       <organization>Google</organization>
       <roles>
         <role>Developer</role>
       </roles>
     </developer>
     <developer>
-      <id>ajaykannan</id>
-      <name>Ajay Kannan</name>
-      <email>ajaykannan@google.com</email>
+      <id>pongad</id>
+      <name>Michael Darakananda</name>
+      <email>pongad@google.com</email>
       <organization>Google</organization>
       <roles>
         <role>Developer</role>
       </roles>
     </developer>
     <developer>
-      <id>ozarov</id>
-      <name>Arie Ozarov</name>
-      <email>ozarov@google.com</email>
+      <id>shinfan</id>
+      <name>Shin Fan</name>
+      <email>shinfan@google.com</email>
       <organization>Google</organization>
       <roles>
         <role>Developer</role>
       </roles>
     </developer>
     <developer>
-      <id>mziccardi</id>
-      <name>Marco Ziccardi</name>
-      <email>mziccardi@google.com</email>
+      <id>michaelbausor</id>
```

Comment:
According to the description:https://maven.apache.org/pom.html#Developers This developer list should be listing members who are **immediately responsible** for the code and if the person should not be contacted about the project, they need not be listed here.